samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -113,33 +113,49 @@
         @foreach (var node in PagedNodeList.Results)
         {
         <tr>
-            <td>@node.NodeName</td>
-            <td>@node.Id</td>
+            <td class="hover-text width-medium">
+                <div>
+                    @node.NodeName
+                </div>
+            </td>
+            <td class="hover-text width-large1">
```

Comment:
>large1 [](start = 40, length = 6)

is this 1 in "...large1" correct? or should it be just "...large"

Reply:
it's correct, but I rename it